docker-compose.yml

```diff
@@ -28,9 +30,22 @@ services:
     build: ./
     environment:
       - CELERY_BROKER_URL=amqp://guest@rabbitmq//
-    command: celery worker -A AIPscan.worker.celery --loglevel=info
+      - TYPESENSE_HOST=typesense
+      - TYPESENSE_API_KEY=xyz
+    command: celery -A AIPscan.worker.celery worker --loglevel=info
     volumes:
       - "./:/src:rw"
     depends_on:
       - rabbitmq
       - aipscan
+
+  typesense:
+    image: typesense/typesense:0.26.0.rc23
+    environment:
+      TYPESENSE_DATA_DIR: /data
+      TYPESENSE_API_KEY: xyz
+    volumes:
+      - /tmp/typesense-server-data:/data
```

Comment:
That directory may not exist in other environments, I'd use a named volume like the ones for RabbitMQ.

Reply:
I've approved, but I'd change this before merging @mcantelon.

Reply:
Done (very cool feature those named volumes)!